src/index.js

```diff
@@ -0,0 +1,227 @@
+function sumIfEven (a, b) {
+    if (a%2==0) {
+        return a*b
+    } else return a+b
+};
+
+function getQuarter (x, y) {
+    if ( x === 0 && y === 0 ) {
+        return "origin";
+    } else if ( x === 0 && y === 1 ) {
+        return "y-axis";
+    } else if ( x===1 && y===0 ) {
+        return "x-axis"
+    } else if ( x>0 && y>0 ) {
+        return "first quarter"
+    } else if ( x>0 && y<0 ) {
+        return "second quarter"
+    } else if ( x<0 && y<0 ) {
+        return "third quarter"
+    } else if ( x<0 && y>0 ) {
+        return "forth quarter"
+    }
+
+};
+
+function addPositives (a,b,c) {
+    let result = 0 ;
+    if (a>0) {
+        result +=a
+    }
+    if (b>0) {
+        result +=b
+    }
+    if (c>0) {
+        result +=c
+    }
+    return result 
+ };
+
+ function calculateMax (a,b,c) {
+        let sum = a + b + c;
+        let product = a * b * c;
+        let result = 0;
+        
+        if(sum === product) {
+            result = 3;
+        }
+        if(sum > product) {
+            result = sum + 3;
+        } else result = product + 3;
+        
+        return result;
+    };
+
+const getMark = rate =>  {
+     let result = "" ;
+     
+     if (rate <0 || rate > 100) {
+         return result ;
+     } else if (rate > 0 && rate < 20) {
+         result = "F"
+     } else if (rate < 40) {
+         result = "E"
+     } else if (rate < 60) {
+        result = "D"
+     } else if (rate < 75) {
+         result = "C"
+     } else if (rate < 90) {
+         result = "B"
+     } else if (rate <= 100) {
+         result = "A"
+     } 
+        return result;
+ };
+
+ const getPrime = number => {
+    for (let i = 2; i < number ; i++) {
+        if (number % i===0) {
+            return false;
+        } else return true;
```

Comment:
This function will not iterate over all needed dividers. It will exit after the first one in any case.